pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/render.go

```diff
@@ -270,7 +272,10 @@ func (r *components) renderComponent(ctx context.Context, acc v1alpha2.Applicati
 		}
 	}
 	// set the owner reference after its ref is edited
-	w.SetOwnerReferences([]metav1.OwnerReference{*ref})
+	// if application controller set the ownerReference already don't override it
+	if len(w.GetOwnerReferences()) == 0 {
```

Comment:
we shoud check the namespace first, only different namespace should do this, otherwise, we need to override it.

You can record the AC namespace in r *components